src/constants/query.ts

```diff
@@ -19,3 +19,4 @@ export const QUERY_KEYS = {
 
 /** 1 hour */
 export const DEFAULT_STALE_TIME = 3_600_000;
+export const ONE_HOUR_STALE_TIME = 60 * 60 * 1000;
```

Comment:
since both are the same can we remove one and only use one throughout the codebase